ports/windows/.appveyor.yml

```diff
@@ -63,10 +63,7 @@ after_test:
       throw "$env:MSYSTEM build exited with code $LASTEXITCODE"
     }
     cd (Join-Path $env:APPVEYOR_BUILD_FOLDER 'mpy-cross')
-    # Building of mpy-cross hasn't been fixed across all possible windows/WSL/...
-    # variations and the STRIP step tries to strip mpy-cross whereas that should be
-    # mpy-cross.exe. Workaround for now by skipping actual strip and size commands.
-    C:\msys64\usr\bin\bash.exe -l -c "make -B -j4 V=1 STRIP=echo SIZE=echo"
+    C:\msys64\usr\bin\bash.exe -l -c "make -B -j4 V=1"
```

Comment:
@stinos is this the correct change?  Appveyor does pass...

Reply:
Yes that's it. See build output showing this works now:

```
strip  micropython.exe
size $(find build -path "build/build/frozen*.o") micropython.exe
   text	   data	    bss	    dec	    hex	filename
 334148	   6636	  15056	 355840	  56e00	micropython.exe
```

Reply:
Where exactly is that output?  I searched in the Appveyor logs but couldn't find it...

Reply:
Aah, I see it now, in the release builds (not debug), there is:
```
[00:02:17] strip  mpy-cross.exe
[00:02:17] size $(find build -path "build/build/frozen*.o") mpy-cross.exe
[00:02:17]    text         data     bss     dec     hex filename
[00:02:17]  305700         4148    3272  313120   4c720 mpy-cross.exe
```